src/routes/[deviceUID]/dashboard/+page.svelte

```diff
@@ -71,6 +72,10 @@
     history = data.history;
   }
 
+  if (data && data.airIndoors !== undefined && data.airIndoors === 1) {
```

Comment:
simplify this logic now that a boolean is being delivered always for `airIndoors` from the server
```
  if (data && data.isIndoors) {
    hideMap = true;
  }
```